front/src/views/Boxes/components/BoxesTable.tsx

```diff
@@ -71,6 +71,8 @@ function BoxesTable({
   const [refetchBoxesIsPending, startRefetchBoxes] = useTransition();
   const { data: rawData } = useReadQuery(boxesQueryRef);
   const tableData = useMemo(() => boxesRawDataToTableDataTransformer(rawData), [rawData]);
+  const boxesCount = tableData.length;
+  const itemsCount = tableData.reduce((total, row) => total + row.numberOfItems, 0);
```

Comment:
@fhenrich33 The counts should adjust depending on the applied filters (e.g. a certain location, tag, and/or box state; or depending on the search text field).